_layouts/downloads.html

```diff
@@ -294,13 +421,45 @@ <h1 class="title-bg">{{site.data.pages.es.downloads.perftop.title}}</h1>
 
         <!-- Descriptions -->
         <div class="row justify-content-center text-center py-3 px-2">
-          <div class="text-small"> </div>
+          <div class="text-small"></div>
+        </div>
+      </div>
+    </div>
+
+
+    <!-- Misc section -->
+    <div class="row px-3 justify-content-center text-center px-2">
+      <div class="col-12 col-md-12"
+           style="background-color:white; border-style: solid; border-color:#b4bcc2; border-width: 1px;"></br>
+        <div class="link-on-white-paragraph">
+          What's in this release? View the
+          <a href="{{site.data.pages.es.downloads.misc.release_notes}}-{{site.data.pages.es.downloads.version.odfe}}.md"
+             target="about:blank"
+             onclick="trackOutboundLink('{{site.data.pages.es.downloads.misc.release_notes}}-{{site.data.pages.es.downloads.version.odfe}}.md');">
+            <h7 style="font-weight:bold;">release notes</h7>
+          </a>
+        </div>
+        <div class="link-on-white-paragraph">
+          Are you looking for a different version? View
+          <a href="{{site.data.pages.es.downloads.misc.previous_releases}}"
```

Comment:
`Looking for an older version? View <a href="{{site.data.pages.es.downloads.misc.previous_releases}}.`

Reply:
I have made the suggested changes.